code/components/citizen-server-impl/include/state/SyncTrees_Five.h

```diff
@@ -1023,7 +1023,74 @@ struct CDoorScriptInfoDataNode { bool Parse(SyncParseState& state) { return true
 struct CDoorScriptGameStateDataNode { bool Parse(SyncParseState& state) { return true; } };
 struct CHeliHealthDataNode { bool Parse(SyncParseState& state) { return true; } };
 struct CHeliControlDataNode { bool Parse(SyncParseState& state) { return true; } };
-struct CObjectCreationDataNode { bool Parse(SyncParseState& state) { return true; } };
+
+struct CObjectCreationDataNode {
+	uint32_t m_model;
+
+	bool Parse(SyncParseState& state)
+	{
+		int unk0 = state.buffer.Read<int>(5);
+		if (unk0 & 0xFFFFFFFD)
+		{
+			uint32_t model = state.buffer.Read<uint32_t>(32);
+			m_model = model;
+
+			bool unk2 = state.buffer.ReadBit();
+			bool unk3 = state.buffer.ReadBit();
+			bool unk4 = state.buffer.ReadBit();
+
+			if (unk3)
+			{
+				auto unk5 = state.buffer.Read<int>(19);
```

Comment:
Script Grab Position - NOT A SINGLE VALUE READ AND WRONG! (as seen by use of method +152d) is a vector3 read with the usual project-specific divisor.

See [an example](https://github.com/citizenfx/fivem/blob/3b1fad02810dc44674314185da1de3450c3ca388/code/components/citizen-server-impl/include/state/SyncTrees_Five.h#L1274). This ought to be shared somewhere in the future.